### ISSUE TITLE
no runtime log with qpython 1.1.1

### ISSUE THREAD

**Issue by llapalus** [qpython-android/qpython.org#82]
Hi,
I'm new to python and kivy, but it seems the runtime log is missing with the last upgrade.
I'm running qpython on my samsung phone and my transformer tablet and now I always get "No log information" in the log, instead of the DEBUG, INFO, etc. informations.

I tried to change the information level with these lines but it didn't change a thing:
  import logging
  logging.getLogger().setLevel(logging.DEBUG)

I also tried to change the #qpy:kivy with #qpy:qpyapp but all I get is the message:
KeyError: 'ANDROID_APP_PATH'

Is there a way to have it back ? If not, is there a repository with the previous versions of qpython ?

Thanks,
Laurent

**Comment by Vagen** [comment]
I am getting the same thing 

Roderick

**Comment by idiot-z** [comment]
Hi,
I'm having the same problem.
As temporary workaround, I added
```
import logging
logging.basicConfig(filename='/path/to/your/log.log')
```
at the top of my *main.py*
You can then view the log file with an external viewer (such as firefox).
Hope that helps.

**Comment by riverfor** [comment]
llapalus, do you run your kivy project from editor ? The "No log information"  is a bug, can you ignore it?
I will remove it in next update.

If you were looking the old versions, please visit http://www.pgyer.com/qpython

**Comment by riverfor** [comment]
idiot-z , thanks for giving the temporary solution.

**Comment by llapalus** [comment]
@riverfor: yes, i'm running it from the editor with the triangle icon. All my .py and .kv files are in a subdir under the directory project.

@idiot-z: thanks. I had to redirect stderr and stdout to get my error messages, basicConfig wasn't enough.
import sys
f=open('.run.log','w') 
f.close()
sys.stdout=open('.run.log','a') 
sys.stderr=open('.run.log','a')

**Comment by llapalus** [comment]
Fixed with 1.2 release. Thanks.

Closing.
